src/pc/tracker.rs

```diff
@@ -85,6 +86,14 @@ pub struct SubxtTracker {
 	last_relay_block_ts: Option<u64>,
 	/// Session tracker
 	session_data: Option<SubxtSessionTracker>,
+	/// Known inbound HRMP channels, indexed by source parachain id
+	inbound_hrmp_channels: Vec<u32>,
```

Comment:
Do we really need this separately if we store them as keys in the `_messages BTreeMap` ?